### DISCUSSION TITLE
Displayname trigger

### DISCUSSION THREAD

**Discussion by thefab86** [ikemen-engine/Ikemen-GO#916] [Feature Request] [answered]
There are name trigger and authorname triggers. What about a displayname trigger?

**Comment by potsmugen** [comment]
That reminds me a sctrl to change the displayname would be pretty neat. Imagine changing the character's name for alternate modes or transformations.

**Comment by thefab86** [comment]
Yeah, I'm working on multilanguage characters, and obviously displayname changes from a language to another, so it could be useful to trigger some images or sounds depending from the displayname.

**Comment by K4thos** [comment, marked as answer]
added in #1608
